CHANGELOG.md

```diff
@@ -97,6 +97,23 @@
 ### ⬆️ Improved
 
 ### ✅ Added
+- Created `FirebaseMessagingDelegate` that help you to build your own `FirebaseMessagingService` implementation
```

Comment:
We should move the code from here to the Firebase docs page, and link to it in the release notes

We can probably still keep all the current code there, just also show this as a simpler alternative